lib/.npmignore

```diff
@@ -0,0 +1,3 @@
+src/
+!/dist/types/src
+dist/tsconfig.tsbuildinfo
```

Comment:
Better to exclude the development related files as well like,

tsconfig.json
webpack.config.js
babel.config.js
.eslintrc.js
.eslintignore

Reply:
Done with the changes